README.md

```diff
@@ -30,8 +30,8 @@ When creating a Pull Request from the forked repository, the default base reposi
 2. Run `npm start`. This will build the app and watch for changes.
 3. In a different terminal, run `npm run serve`
 4. Go to app.unbounce.com and create a Smart Builder page on the client where local development has been enabled.
-5. Open the apps sidebar panel and scroll down until you see "Add your own extension". Click "Add Manifest"
-6. Copy and paste the `manifest.json` file from this repo into the box.
+5. Open the apps sidebar panel and scroll down until you see "Add your own extension".
+6. Set the port your app is running on, by default this is 8081.
```

Comment:
It doesn't prompt for the port. It lets you add the app automatically, which is great. But it doesn't match with this doc.

Reply:
Good catch. I wrote these docs before the UX updates. Will update 👍